src/components/EnemyStatsDisplay.js

```diff
@@ -1,57 +1,67 @@
-import {StyleSheet, Text, View, Image} from 'react-native';
+import {StyleSheet, Text, View, Image, FlatList} from 'react-native';
 import React from 'react';
+import atkIcon from '../assets/images/icon-sword.png';
+import defIcon from '../assets/images/icon-shield.png';
+import hpIcon from '../assets/images/icon-heart.png';
 
-const EnemyStatsDisplay = () => {
+const EnemyStatsDisplay = ({enemy}) => {
+  const addEquipmentStats = char => {
+    const stats = Object.assign({}, char);
+    stats.equipment.forEach(
+      equipment =>
+        (stats[equipment.affected_attribute] += equipment.affected_amount),
+    );
+    return stats;
+  };
+  const enemyInfo = addEquipmentStats(enemy);
+  console.warn(enemy);
+  // const enemyStats = [
```

Comment:
It's a good practice to avoid pushing commented code, if you want this code in future, git will keep a copy of that for you, for now i think it's better remove this